projects/igniteui-angular/src/lib/grids/grid-base.directive.ts

```diff
@@ -6523,7 +6524,7 @@ export abstract class IgxGridBaseDirective extends DisplayDensityBase implements
         let res = !this.nativeElement.parentElement ||
             this.nativeElement.parentElement.clientHeight === 0 ||
             this.nativeElement.parentElement.clientHeight === renderedHeight;
-        if (!isChromium()) {
+        if (!isChromium() && !isFireFox()) {
```

Comment:
I think the grid should use the `PlatformUtil` rather than to have separate implementation for `isChromium` or `isFireFox`.